crates/hyperqueue/src/client/commands/worker.rs

```diff
@@ -143,21 +165,32 @@ pub async fn start_hq_worker(
 pub async fn get_worker_list(
     connection: &mut ClientConnection,
     include_offline: bool,
```

Comment:
```suggestion

```
This parameter should be removed. If `--all` nor `--filter` is entered, the command code should pass `Some(WorkerFilter::Running)`.